Util/Docker/.dockerignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
New to docker: This file is excluded from your build context, because you save intermediate builds here? Or is this the location where you store your final "output build"?

Reply:
Yes that's an artifact of my developing process. Can be removed.

I did copy the whole simulator to this build directory on my dev machine. That way I was able to test if the simulator launches as expected. Afterwards I managed to run the simulator inside a Docker container, so the build folder became obsolete.

Reply:
Okay, thanks. So this doesn't hurt anyone and we just keep it?

Reply:
I mean it's doing nothing meaningful and just adds up more complexity, so better get rid of it ^^